### DISCUSSION TITLE
CarvingConfig.txt

### DISCUSSION THREAD

**Discussion by gilmaralves** [sepinf-inc/IPED#1467] [Q&A] [answered]
In IPEDConfig.txt, there's a reference to the file "conf/CarvingConfig.txt", but this file don't exist. Was "conf/CarvingConfig.txt" replaced by the file CarverConfig.xml?? 

If so, I can ignore the following message:
"[ERROR] [task.carver.CarverTask]                        addUnallocated is disabled, so carving will NOT be done in unallocated space!"

**Comment by wladimirleite** [comment, marked as answer]
Hi @gilmaralves!

> In IPEDConfig.txt, there's a reference to the file "conf/CarvingConfig.txt", but this file don't exist. Was "conf/CarvingConfig.txt" replaced by the file CarverConfig.xml??

It is a typo. While working on #1344 I noticed and fixed that. It means CarverConfig.**xml**.
 
> If so, I can ignore the following message: "[ERROR] [task.carver.CarverTask] addUnallocated is disabled, so carving will NOT be done in unallocated space!"

Carving is done in a few types of items, and unallocated space is probably the most important of them.
This message means that you enabled carving but did not enable **addUnallocated**, which is usually not the intended behavior.
You can change this setting in **conf/FileSystemConfig.txt**.

**Comment by gilmaralves** [comment]
Hi, @tc-wleite 

thanks for the answer.

**Comment by antomiccoli** [comment]
but the file "conf/carvingconfig.txt" does not exist?
How do I select the file types to carve?
Thank you

**Reply by patrickdalla** [reply]
conf/carvingconfig.txt does not exists.

You remove or add carving types editing CarvingConfig.xml. 

P.S. In 4.2 there will be an UI to help in config. where you can unselect the carving types you don't want and create a new profile. It would be helpful if you test.

**Reply by antomiccoli** [reply]
OK.
Where I find the 4.2 version?

**Reply by antomiccoli** [reply]
On the Web Site there is only the 4.1.6 version.

**Reply by patrickdalla** [reply]
Sorry. "there will be" so it will be released yet. Final testing fases. You can download the last master snapshot and will help in testing.

I find the snapshot in action github tab of the project, and search for last master push. the link to the snapshot will be at the end of the page. 

I Don't know if there is an easier way to find it.

**Reply by antomiccoli** [reply]
OK, I found it but didn't find the new UI

**Reply by patrickdalla** [reply]
Sorry again. It seems it is not in master yet. In fact, it is in another repo as it was started by another developer. I will try to figure it out.

**Reply by patrickdalla** [reply]
This is the repo: https://github.com/thiagofuer/IPED/. It is on the master banch of this repo.

The workflow to automatically generate a snapshot every push wasn't enabled. Now it is.

Link to last run:
https://github.com/thiagofuer/IPED/actions/runs/10080249279

**Reply by patrickdalla** [reply]
And thank for the interest in help testing.

**Comment by antomiccoli** [comment]
Thank's

Il Mar 23 Lug 2024, 17:37 PATRICK DALLA BERNARDINA ***@***.***>
ha scritto:

**Comment by antomiccoli** [comment]
hi Patrick, thank's for all.
I have a new question, to use the new version of IPED (4.2) i need the IPEDTools.exe 2.1.3 correct?
